src/apis/product.apis.ts

```
@@ -63,6 +71,23 @@ export const getOrders = (
     token
   );
 };
+export const getAllOrders = (
+  token: string,
+  page: number = 1,
+  status?: "PENDING" | "IN_TRANSIT" | "COMPLETED" | "MERCHANT_ACCEPTED"
+) => {
+  let queryParams = "";
+  if (!!status) {
```

Biome

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)